doc/element_primary_key.tex

```diff
@@ -9,7 +9,14 @@
 
 The type of the key must always be specified by the \texttt{@dmtype} attribute. 
 
-\begin{lstlisting}[caption={The \texttt{INSTANCE} is identified within a \texttt{COLLECTION} by the \texttt{PRIMARY\_KEY} value (see \ref{PRIMARY_KEY_snippet}).},language=XML]
+\begin{lstlisting}[caption={The \texttt{INSTANCE} is identified within a \texttt{COLLECTION} by the \texttt{PRIMARY\_KEY} value (see line~\ref{PRIMARY_KEY_snippet} in Appendix~\ref{appendix_A}).},language=XML]
```

Comment:
It looks like there is a duplicate lstlisting here with the same content.

Reply:
fixed